files/en-us/web/api/htmlmapelement/name/index.html

```diff
@@ -0,0 +1,46 @@
+---
+title: "HTMLMapElement: name property"
+short-title: name
+slug: Web/API/HTMLMapElement/name
+page-type: web-api-instance-property
+browser-compat: api.HTMLMapElement.name
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`name`** property of the {{domxref("HTMLMapElement")}} is a string gives the map unique name. 
+This property is used for referencing `useMap` attribute of the {{HTMLElement("img")}} element. 
+The `name` property can not be set to empty or with whitespaces. 
+If `id` is given to a {{HTMLElement("map")}} element then `name` property should be same as `id`.
```

Comment:
`id` was not defined, I think you want to speak about the reflection.

```suggestion
This property reflects the `name` attribute of the {{HTMLElement("map")}} element.
```

Reply:
Yes, agreed.